package.json

```diff
@@ -54,5 +54,8 @@
   },
   "engines": {
     "node": ">=4"
+  },
+  "dependencies": {
+    "bootstrap": "^5.1.0"
```

Comment:
I think it's not good to install `bootstrap` as a dependency.

Reply:
Okay, fair enough.